examples/java/src/test/java/dev/selenium/getting_started/SetupAndTeardownTest.java

```diff
@@ -0,0 +1,78 @@
+package dev.selenium.getting_started;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.Duration;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class SetupAndTeardownTest {
+
+	private WebDriver driver;
+	private WebDriverWait wait;
+	private By projectsLink = By.xpath("//a/span[text()='Projects']");
+	private By searchFieldSpan = By.xpath("//span[text()='Search']");
+	private By searchField = By.id("docsearch-input");
+	private By searchResultsFirstItem = By.cssSelector("ul[role='listbox'] li");
+
+	@BeforeAll
+	public static void beforeAll() {
```

Comment:
The script we want to show exists already: https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/examples/java/src/test/java/dev/selenium/getting_started/UsingSeleniumTest.java#L16.

Could you please link the existing one instead? Feel free to modify it to have the setup and tear down.

Reply:
![image](https://github.com/SeleniumHQ/seleniumhq.github.io/assets/75480921/a98d0e4a-76c0-4aaa-8e2f-cb05d8dfbc89)

Actually by reading this line, I implied that for Assertion and Setup / Teardown examples, we should be creating different files.
So based on the above comment, should I remove the SetupAndTeardownTest.java and AssertionsTest.java ? In that case, we can add the setup and teardown code in UsingSelenium.java and assertions are already being used in that.
 
Kindly let me know how you want to do it, I will make the appropriate changes.

Reply:
Just using the `UsingSelenium.java` is enough. Thank you.

Reply:
Got it, thanks for the clarification. Making changes and raising the PR.